functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -26,10 +26,16 @@
 )
 from sdcm.utils.decorators import log_run_info, retrying
 from sdcm.utils.k8s import HelmValues
+from sdcm.utils.sstable.load_inventory import MULTI_COLUMNS_DATA, BIG_SSTABLE_MULTI_COLUMNS_DATA
 from sdcm.wait import wait_for
 
 log = logging.getLogger()
 
+FUNCTIONAL_TEST_DATASETS = {"small": MULTI_COLUMNS_DATA,
```

Comment:
I would ditch this mapping, and use the name directly

```yaml
k8s_scylla_operator_functional_test_dataset:  BIG_SSTABLE_MULTI_COLUMNS_DATA
```

```
import sdcm.utils.sstable.load_inventory as datasets

test_data = getattr(datasets, dataset_name, None)
```

so it would be clearer and searchable, and no need keep that mapping in mind, when adding new, or when jumping between the configuration yaml, to the actually usage.

Reply:
changed